host/xtest/aes_gcm_counter_overflow_test_decrypt.h

```diff
@@ -0,0 +1,365 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * Copyright 2024 NXP
+ */
+
+{ TEE_ALG_AES_GCM, TEE_MODE_DECRYPT, TEE_TYPE_AES,
+/* Key */ (const uint8_t []){ 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff, }, 16,
+/* IV  */ (const uint8_t []){ 0xf9, 0x5f, 0xde, 0x4a, 0x75, 0x19, 0x13, 0x20, 0x2a, 0xee, 0xee, 0x32, 0xa0, 0xb5, 0x57, 0x53, }, 16,
+0,
+/* AAD */ NULL, 0,
+0,
+/* PT  */ (const uint8_t []){ 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, }, 40,
+/* CT  */ (const uint8_t []){ 0x00, 0x07, 0x8d, 0x10, 0x9d, 0x92, 0x14, 0x3f, 0xcd, 0x5d, 0xf5, 0x67, 0x21, 0xb8, 0x84, 0xfa, 0xc6, 0x4a, 0xc7, 0x76, 0x2c, 0xc0, 0x9e, 0xea, 0x2a, 0x3c, 0x68, 0xe9, 0x2a, 0x17, 0xbd, 0xb5, 0x75, 0xf8, 0x7b, 0xda, 0x18, 0xbe, 0x56, 0x4e, }, 40,
+/* Tag */ (const uint8_t []){ 0x15, 0x2a, 0x65, 0x04, 0x5f, 0xe6, 0x74, 0xf9, 0x76, 0x27, 0x42, 0x7a, 0xf5, 0xbe, 0x22, 0xda, }, 16,
+77},
```

Comment:
Could you also add a space char before close braces.  I think it would be better to have clear new-line chars between sub case data blocks, like:
```c
{ TEE_ALG_AES_GCM, TEE_MODE_DECRYPT, TEE_TYPE_AES,
   <all data>
},
{ TEE_ALG_AES_GCM, TEE_MODE_DECRYPT, TEE_TYPE_AES,
   <all data>
},
...
```

Sorry for the noise but since you use a nice script to produce these arrays I think it could help new readers and maintenance to have explicit `struct xtest_ae_case` field names:
```c
{
	.algo = TEE_ALG_AES_GCM, .mode = TEE_MODE_DECRYPT, .key_type = TEE_TYPE_AES,
	.key = (const uint8_t []){ ... },
	.key_len = <value>,
	.nonce = (const uint8_t []){ ... },
	.nonce_len = <value>,
	.aad_incr = <value>,
	.aad = (const uint8_t []){ ... },
	.aad_len = <value>,
	.in_incr = <value>,
	.ptx = (const uint8_t []){ ... },
	.ptx_len = <value>,
	.ctx = (const uint8_t []){ ... },
	ctx_len = <value>,
	.tag = (const uint8_t []){ ... },
	.tag_len = <value>,
},
```
Feel free to ignore this comment if you think it's not worth it.

Reply:
it worth it  :+1: